mesheryctl/internal/cli/root/system/fixtures/.meshery/TestContext.yaml

```diff
@@ -3,21 +3,21 @@ contexts:
     endpoint: http://localhost:9081
     token: Default
     platform: kubernetes
-    adapters:
-    - meshery-istio
+    components:
+      - meshery-istio
```

Comment:
@piyushsingariya, this extra space will not cause any issue?

Reply:
Nope. It won't. Else tests would have failed.